src/ucp/core/ucp_worker.c

```diff
@@ -2476,6 +2477,24 @@ ucs_status_t ucp_worker_query(ucp_worker_h worker,
     return status;
 }
 
+ucs_status_t ucp_worker_address_query(ucp_address_t *address,
+                                      ucp_worker_address_attr_t *attr)
+{
+    void *p = address;
+    uint8_t address_header;
+
+    if (attr->field_mask & UCP_WORKER_ADDRESS_ATTR_FIELD_UUID) {
+        address_header = *ucs_serialize_next(&p, const uint8_t);
+        if (!(address_header & UCP_ADDRESS_HEADER_FLAG_UUID)) {
+            return UCS_ERR_INVALID_PARAM;
```

Comment:
maybe return some special "undefined uuid"?

Reply:
you mean in addition set uuid to fffffffffffffff for instance?

Reply:
yes, or some other status like NO_ELEM